File: app/src/main/java/com/smedialink/abakarmagomedov/githubapplication/presentation/UsersActivity/UsersPresenter.java
package com.smedialink.abakarmagomedov.githubapplication.presentation.UsersActivity;

/**
 * Created by abakarmagomedov on 15/05/17.
 */

public interface UsersPresenter {

    void getUsers();
    void attachView(UsersView view);
    void detachView();

}

File: app/src/main/java/com/smedialink/abakarmagomedov/githubapplication/presentation/UsersActivity/UsersPresenter.kt
package com.smedialink.abakarmagomedov.githubapplication.presentation.UsersActivity

/**
 * Created by abakarmagomedov on 15/05/17.
 */
interface UsersPresenter {
    val users: Unit
    fun attachView(view: UsersView)
    fun detachView()
}
